ui/components/component-library/select-button/select-button.tsx

```diff
@@ -11,52 +11,100 @@ import {
   BorderColor,
   BorderRadius,
   Display,
+  FlexDirection,
   JustifyContent,
+  TextColor,
+  TextVariant,
 } from '../../../helpers/constants/design-system';
 import {
   SelectButtonProps,
   SelectButtonComponent,
   SelectButtonSize,
 } from './select-button.types';
 
+// Utility function to check for plain objects
+const isPlainObject = (obj: any) => {
```

Comment:
The `isPlainObject` function takes an argument of type `any`. It's generally recommended to avoid using any when possible as it can lead to less type-safe code. If you expect this function to receive an object, you could use `unknown` or a more specific type?